packages/docs/src/components/package-manager-tabs/index.tsx

```diff
@@ -0,0 +1,154 @@
+import type { CookieOptions } from 'express';
+import { $, Slot, component$, useContext, useTask$ } from '@builder.io/qwik';
+import { Tab, TabList, TabPanel, Tabs } from '@qwik-ui/headless';
+import { PnpmIcon } from './pnpm';
+import { YarnIcon } from './yarn';
+import { NpmIcon } from './npm';
+import { BunIcon } from './bun';
+import { GlobalStore } from '../../context';
+import { isBrowser, isServer } from '@builder.io/qwik/build';
+
+export type PackageManagers = 'npm' | 'yarn' | 'pnpm' | 'bun';
+
+const getCookie = (name: string) => {
+  const cookieName = name + '=';
+  const cookies = decodeURIComponent(document.cookie).split(';');
+  for (let i = 0; i < cookies.length; i++) {
+    let cookie = cookies[i];
+    while (cookie.charAt(0) == ' ') {
+      cookie = cookie.substring(1, cookie.length);
+    }
+    if (cookie.indexOf(cookieName) == 0) {
+      return cookie.substring(cookieName.length, cookie.length);
+    }
+  }
+  return null;
+};
+
+const setCookie = (name: string, value: string, options: CookieOptions = {}) => {
```

Comment:
Can you read cookie inside routeLoader$ pls?
--
We already have a setCookie method here `packages/docs/src/routes/(shop)/utils.ts`.
Can you create a common method to manage both cases pls?

Reply:
i will make those changes by tomorrow.

Reply:
Still relevant.